index.js

```diff
@@ -793,6 +611,40 @@ app.post(
   },
 )
 
+app.post('/api/startChatbot', Validator.body(['coreid', 'event']).exists(), async (request, response) => {
+  try {
+    const validationErrors = Validator.validationResult(request).formatWith(helpers.formatExpressValidationErrors)
+
+    if (validationErrors.isEmpty()) {
+      const coreId = request.body.coreid
+      const alertType = request.body.event.split(' ')[0]
+      helpers.log(alertType)
```

Comment:
Do you still want to keep this log? For it to be useful, it could use some more info about the location, I think.